app/controllers/kadmin/application_controller.rb

```diff
@@ -57,30 +57,18 @@ def handle_error(error, options = {})
     #
     # organization_scoped_ar is an ActiveRecord that has organization_scope(Organization) scope defined
     def scoped_find_by!(organization_scoped_ar, id)
-      if authorized_user.admin?
-        if id.is_a?(Array)
-          return organization_scoped_ar.find(id)
-        else
-          return organization_scoped_ar.find_by!(id: id)
-        end
+      if id.is_a?(Array)
```

Comment:
You dont really need check if id is []. This is enough:
```organization_scoped_ar.find(id)```

Reply:
done, thanks for the review!